Create Language-and-script-of-the-description.md

jess-grz opened the pull request:
Will need to edit links and add new ones, especially in 0.4.1.3.

jess-grz commented:
Perfect, thanks for all the helpful feedback, Kalan! I'm still trying to
wrap my head around the new numbering and linking to other chapters that
aren't posted yet. I'm glad the X.X. citation method works for now. It was
pretty fun to try this out, and I'm happy to work on some other sections
when they're ready.

On Tue, Jul 21, 2020 at 10:12 AM kkdavis <notifications@github.com> wrote:

> *@ladylazarus3* commented on this pull request.
>
> Jessica,
>
> This looks really great! Thank you so much for doing this work. Overall,
> your formatting was absolutely perfect. the only things i changed are i
> changed the numbering in the (e.g.) in 0F1.2/0.4.1.3 to the new numbering
> system and added some external links to the ALA Romanization Tables.
>
> i really like how you added X.X to the places where we know the chapter
> and entity numbers, but not the section or instruction number as i think we
> can utilize the website search feature to locate X.X citations in final
> "DCRMR polishing" steps and replace them with the real numbering as the
> pages they reference are finalized in the future.
>
> This one is a wrap for now. If some of my changes are wonky or if you have
> comments on the changes, either pull another request or email me. We are
> all still figuring this out (me included). Thanks again, :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rbms-bsc/DCRMR/pull/37#pullrequestreview-452560078>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKRS7NULCJOO2ME66VJR7MDR4WV7XANCNFSM4PC5UVAA>
> .
>

ladylazarus3 commented:
Well,  funny you should mention that... :) :) The order of the text
spreadsheet
<https://docs.google.com/spreadsheets/d/1nP4uJPxj6gbfWPcPusKxAqxBXbQW9HG8uoU1SxZMqq0/edit#gid=820222028>
for
the various preliminary sections has all lots and lots of Ys in the "Text
Finalized" column.  If you want to go for it, i would say go for it.  just
make sure that you sign your name on the "who transferred?" column.  so
that we don't accidentally have two folks working on the same page at once.

Also, make sure to check out the "live page" so you can bask in your
awesomeness!  You made it! :)
https://rbms-bsc.github.io/DCRMR/books/general-rules/Language-and-script-of-the-description/


kalan



On Tue, Jul 21, 2020 at 10:19 AM Jessica Grzegorski <
notifications@github.com> wrote:

> Perfect, thanks for all the helpful feedback, Kalan! I'm still trying to
> wrap my head around the new numbering and linking to other chapters that
> aren't posted yet. I'm glad the X.X. citation method works for now. It was
> pretty fun to try this out, and I'm happy to work on some other sections
> when they're ready.
>
> On Tue, Jul 21, 2020 at 10:12 AM kkdavis <notifications@github.com> wrote:
>
> > *@ladylazarus3* commented on this pull request.
> >
> > Jessica,
> >
> > This looks really great! Thank you so much for doing this work. Overall,
> > your formatting was absolutely perfect. the only things i changed are i
> > changed the numbering in the (e.g.) in 0F1.2/0.4.1.3 to the new
> numbering
> > system and added some external links to the ALA Romanization Tables.
> >
> > i really like how you added X.X to the places where we know the chapter
> > and entity numbers, but not the section or instruction number as i think
> we
> > can utilize the website search feature to locate X.X citations in final
> > "DCRMR polishing" steps and replace them with the real numbering as the
> > pages they reference are finalized in the future.
> >
> > This one is a wrap for now. If some of my changes are wonky or if you
> have
> > comments on the changes, either pull another request or email me. We are
> > all still figuring this out (me included). Thanks again, :)
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/rbms-bsc/DCRMR/pull/37#pullrequestreview-452560078>,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AKRS7NULCJOO2ME66VJR7MDR4WV7XANCNFSM4PC5UVAA
> >
> > .
> >
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/rbms-bsc/DCRMR/pull/37#issuecomment-661925100>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD3BAXHDXS56JPAN5UDQIA3R4WWXJANCNFSM4PC5UVAA>
> .
>


-- 
kalan Knudson Davis, MLIS, Albertian Order of Saint Leibowitz | Special
Collections Metadata Librarian | University of Minnesota Libraries | Data
and Technology Division | Data Management & Access (DMA) Department | 160
Wilson Library | 309 19th Ave. S. | Minneapolis, MN 55455 | (218) 779-9009
Cell | kkdavis@umn.edu | www.linkedin.com/in/kalan-the-librarian |
ORCID: 0000-0002-1032-6042
<https://orcid.org/0000-0002-1032-6042>|
http://id.loc.gov/authorities/names/no2019115914

jess-grz commented:
Sounds good! 🤗 I'll start on another one this week.

On Tue, Jul 21, 2020 at 10:26 AM kkdavis <notifications@github.com> wrote:

> Well, funny you should mention that... :) :) The order of the text
> spreadsheet
> <
> https://docs.google.com/spreadsheets/d/1nP4uJPxj6gbfWPcPusKxAqxBXbQW9HG8uoU1SxZMqq0/edit#gid=820222028
> >
> for
> the various preliminary sections has all lots and lots of Ys in the "Text
> Finalized" column. If you want to go for it, i would say go for it. just
> make sure that you sign your name on the "who transferred?" column. so
> that we don't accidentally have two folks working on the same page at once.
>
> Also, make sure to check out the "live page" so you can bask in your
> awesomeness! You made it! :)
>
> https://rbms-bsc.github.io/DCRMR/books/general-rules/Language-and-script-of-the-description/
>
>
> kalan
>
>
>
> On Tue, Jul 21, 2020 at 10:19 AM Jessica Grzegorski <
> notifications@github.com> wrote:
>
> > Perfect, thanks for all the helpful feedback, Kalan! I'm still trying to
> > wrap my head around the new numbering and linking to other chapters that
> > aren't posted yet. I'm glad the X.X. citation method works for now. It
> was
> > pretty fun to try this out, and I'm happy to work on some other sections
> > when they're ready.
> >
> > On Tue, Jul 21, 2020 at 10:12 AM kkdavis <notifications@github.com>
> wrote:
> >
> > > *@ladylazarus3* commented on this pull request.
> > >
> > > Jessica,
> > >
> > > This looks really great! Thank you so much for doing this work.
> Overall,
> > > your formatting was absolutely perfect. the only things i changed are i
> > > changed the numbering in the (e.g.) in 0F1.2/0.4.1.3 to the new
> > numbering
> > > system and added some external links to the ALA Romanization Tables.
> > >
> > > i really like how you added X.X to the places where we know the chapter
> > > and entity numbers, but not the section or instruction number as i
> think
> > we
> > > can utilize the website search feature to locate X.X citations in final
> > > "DCRMR polishing" steps and replace them with the real numbering as the
> > > pages they reference are finalized in the future.
> > >
> > > This one is a wrap for now. If some of my changes are wonky or if you
> > have
> > > comments on the changes, either pull another request or email me. We
> are
> > > all still figuring this out (me included). Thanks again, :)
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/rbms-bsc/DCRMR/pull/37#pullrequestreview-452560078
> >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AKRS7NULCJOO2ME66VJR7MDR4WV7XANCNFSM4PC5UVAA
> > >
> > > .
> > >
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/rbms-bsc/DCRMR/pull/37#issuecomment-661925100>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AD3BAXHDXS56JPAN5UDQIA3R4WWXJANCNFSM4PC5UVAA
> >
> > .
> >
>
>
> --
> kalan Knudson Davis, MLIS, Albertian Order of Saint Leibowitz | Special
> Collections Metadata Librarian | University of Minnesota Libraries | Data
> and Technology Division | Data Management & Access (DMA) Department | 160
> Wilson Library | 309 19th Ave. S. | Minneapolis, MN 55455 | (218) 779-9009
> Cell | kkdavis@umn.edu | www.linkedin.com/in/kalan-the-librarian |
> ORCID: 0000-0002-1032-6042
> <https://orcid.org/0000-0002-1032-6042>|
> http://id.loc.gov/authorities/names/no2019115914
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rbms-bsc/DCRMR/pull/37#issuecomment-661929487>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKRS7NQ2IWV56NYVK6DKFNDR4WXTXANCNFSM4PC5UVAA>
> .
>
